docs/3.0.0-beta.x/admin-panel/customization.md

```diff
@@ -127,6 +127,15 @@ export { languages, translationMessages };
 With this modification only English and French will be available in your admin
 :::
 
+::: tip
+If you need 4+ languages to show in the dropdown you can change the css by copying the file from cache and customizing the css yourself 
+:::
+cp .cache/admin/src/containers/LocaleToggle/Wrapper.js admin/src/containers/LocaleToggle/Wrapper.js
+edit Wrapper.js:67 "max-height: 192px !important;"  
```

Comment:
Hi thanks for adding this to the documentation but I don't really see the difference with the `Wrapper.js` override...

Reply:
Yeah np, prob no need to add it to the docs